spec/rails_helper.rb

```diff
@@ -1,5 +1,8 @@
-require "simplcov"
-SimpleCov.start
+require "simplecov"
+
+SimpleCov.start do
+  add_filter 'spec/rails_helper.rb'
```

Comment:
does the `add_filter` configuration happen automatically with the gem installation?